### ISSUE TITLE
Issues about BulkProcessor with JAVA High Level Client 

### ISSUE THREAD

**Issue by NikolaDai** [elastic/elasticsearch#36713]
i was trying to use bulkprocessor with java high level client. I copied the demo code in [https://www.elastic.co/guide/en/elasticsearch/client/java-rest/current/java-rest-high-document-bulk.html ](https://www.elastic.co/guide/en/elasticsearch/client/java-rest/current/java-rest-high-document-bulk.html ).However, it tells that it can't work and shows that there is no request added to bulkrequest.  then i add one more line of codes as follows
``````
bulkRequest.add(new IndexRequest("articles", "article", String.valueOf(j)).source(lineTxt, XContentType.JSON));
bulkProcessor.add(new IndexRequest("articles", "article", String.valueOf(j)).source(lineTxt, XContentType.JSON));
``````
then it works. i dont why? it is supposed that only use bulkProcessor.add which will lead to the bulkRequest's addition of new request. Anyone helps?

**Comment by albertzaharovits** [comment]
@NikolaDai Can you please share the code that is not working as expected? I find it hard to follow if you're using the bulk processor or the bulk request.

**Comment by javanna** [comment]
hi @NikolaDai please can you ask your question in our forums at https://discuss.elastic.co/ ? We reserve github issues for verified bugs or feature requests.
